### PR TITLE
login style #19

### LINKED ISSUE #19: Login style
#3 
Login style

### PR DESCRIPTION
**Pull request by walaamedhat**
- fixes #19  

- related to #19  

- make login page html,css

### CODE REVIEW

File: public/css/login.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
```
Comment (Salam-Dalloul) [inline public/css/login.css]: why the width is 100%! if you remove it, this will not affect anything :smile:

File: public/css/login.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  background-color: #752223;
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 20px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  position: relative;
```
Comment (Salam-Dalloul) [inline public/css/login.css]: why the position is relative !!

File: public/css/login.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+.h4 {
+  margin-top: 25px;
+  text-align: center;
+}
+
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  align-self: flex-end;
```
Comment (Salam-Dalloul) [inline public/css/login.css]: you have to display flex before using flex properties :smile_cat:

File: public/login.html [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  </header>
+  <div class="container">
+    <img src="./image/logo.png" alt="">
+    <h1 class="h1">Login</h1>
+    <form class="form" action="/login" method="post" id="loginForm">
+      <input class="input" type="text" id='username' placeholder="Username" required>
+      <input class="input" type="password" id='password' placeholder="Password" required>
+      <button class="button" type="submit" name="button" id="btn">Login</button>
+      <h4 class="h4">You don't have an account? <a class="a" href="/register">Sign Up</a></h4>
+      <div id="paragraph">
```
Comment (Salam-Dalloul) [inline public/login.html:22]: why is this paragraph exists!! :rabbit:

**Review by Salam-Dalloul — CHANGES_REQUESTED** [review]
you have to check these ones :dancer:

File: public/login.html (hunk 4 above)
Comment (MahmoudMH) [reply public/login.html:22]: there is tag called p > it's paragraph tag , and if you need it use it , but its empty :-1:

**Review by MahmoudMH — CHANGES_REQUESTED** [review]

File: public/css/login.css (hunk 3 above)
Comment (matthewdking) [reply public/css/login.css]: The line below has `display: flex;` so this is fine :+1:

File: public/css/login.css [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  align-self: center;
+}
+
+.header {
+  width: 100%;
+  height: 40px;
```
Comment (matthewdking) [inline public/css/login.css]: I would advise not using `px` and take a look at `rem` for more responsive site

**Review by matthewdking — CHANGES_REQUESTED** [review]

File: public/css/login.css (hunk 1 above)
Comment (walaamedhat) [reply public/css/login.css]: :+1:

File: public/css/login.css (hunk 2 above)
Comment (walaamedhat) [reply public/css/login.css]: hhhh, I don't understand why i did it  :+1:  I will remove it

File: public/css/login.css (hunk 3 above)
Comment (walaamedhat) [reply public/css/login.css]: Cool, don't effect if this line have before/after >display: flex; ?

File: public/login.html (hunk 4 above)
Comment (walaamedhat) [reply public/login.html:22]: I make this for if i want to show message for user i will put p> inside it

File: public/css/login.css (hunk 5 above)
Comment (walaamedhat) [reply public/css/login.css]: Ok cool i will do it

**Review by Salam-Dalloul — DISMISSED** [review]

**Review by MahmoudMH — DISMISSED** [review]

File: public/css/login.css [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  padding-left: 10px;
+  border-color: #a3a3a3;
+  font-size: 17px;
+}
+
+.button {
+  height: 3.5rem;
+  border-radius: 10px;
+  border: none;
+  height: 55px;
```
Comment (muhmushtaha) [inline public/css/login.css]: duplicated height in the button

File: public/css/login.css [hunk 7]
```diff
@@ -0,0 +1,82 @@
+body {
```
Comment (muhmushtaha) [inline public/css/login.css:1]: run the lint command before pushing any changes

**Review by muhmushtaha — CHANGES_REQUESTED** [review]

**Review by matthewdking — DISMISSED** [review]

File: public/css/login.css (hunk 6 above)
Comment (walaamedhat) [reply public/css/login.css]: i removed it

File: public/css/login.css (hunk 7 above)
Comment (walaamedhat) [reply public/css/login.css:1]: ok

**Review by muhmushtaha — APPROVED** [review]

**Pull request merged**